### PR TITLE
Fix the preview not being updated after editing the image

### PR DESCRIPTION
**Pull request by qborisb**
## Description

Fixes the image preview not being updated after editing the image on `*collection*/*item*` pages

## Type of Change

- [x] Bugfix
- [ ] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

### CODE REVIEW

File: app/src/interfaces/file-image/file-image.vue
Comment (rijkvanzanten) [inline app/src/interfaces/file-image/file-image.vue:137]: Smart to update this to use the timestamp rather than a random hash 👍🏻

File: app/src/interfaces/file-image/file-image.vue
Comment (rijkvanzanten) [inline app/src/interfaces/file-image/file-image.vue:185]: Does this need to "reset" the image as well? Wouldn't just calling `refresh` be enough?

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/interfaces/file-image/file-image.vue
Comment (qborisb) [reply app/src/interfaces/file-image/file-image.vue]: Yes, you are right! Fixed it

**Review by rijkvanzanten — APPROVED** [review]

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]
Also please enable `Allow edits from maintainers.` in the right sidebar on GitHub. We're not able to update the PR to the latest of main, or merge suggestions like the one I just made 👍🏻

**Comment by qborisb** [comment]
@rijkvanzanten I fetched the changes and made the function async. Unfortunately, I cannot `Allow edit from maintainers` because the fork is not from my personal account 😅

**Comment by rijkvanzanten** [comment]
> Unfortunately, I cannot Allow edit from maintainers because the fork is not from my personal account 😅

Odd!

> So, if any further changes are needed on this PR, I will have to create a new fork from my personal account so that I can mark a PR as "editable by maintainers".

Yeah that'd be much appreciated. We often want to merge main right after the last review and before the merge to ensure there's no new conflicts that arise from other changes, and with timezones differences and different availability that process can become a huge PITA if we have to wait for each other back and forth 🙂

**Comment by rijkvanzanten** [comment]
Also for the little tweaks like making the function async, I could've easily done that and merge it through while you were asleep 😄

**Review by rijkvanzanten — APPROVED** [review]
I'll tweak the linter warning on main, as I can't push here 😉

**Comment by qborisb** [comment]
Makes sense @rijkvanzanten, agreed, gonna create a directus fork from a personal account :) Thanks for merging the PR :)

**Pull request merged**
